core/store/models/external_initiator.go

```diff
@@ -19,14 +19,16 @@ type ExternalInitiatorRequest struct {
 
 // ExternalInitiator represents a user that can initiate runs remotely
 type ExternalInitiator struct {
-	*gorm.Model
 	Name           string  `gorm:"not null;unique"`
 	URL            *WebURL `gorm:"url,omitempty"`
 	AccessKey      string  `gorm:"not null"`
 	Salt           string  `gorm:"not null"`
 	HashedSecret   string  `gorm:"not null"`
 	OutgoingSecret string  `gorm:"not null"`
 	OutgoingToken  string  `gorm:"not null"`
+
+	CreatedAt time.Time
```

Comment:
Are these needed?

Reply:
I don't know, they're usually included by default.  I've removed 'ID' and 'DeletedAt' only because the name is the unique ID, and the former was causing problems.

Reply:
I don't think we should be adding code unless we are clear on our motivations for doing so.

Reply:
These fields were carry overs from Gorm's default fields.  They were already in the codebase.  I'm happy to remove them if you are.  I was hesitant because I was leaning towards the converse, removing code should have clear motivations too.

Reply:
Ahh, thank you.